src/RpcClient/StateApi.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Neo.IO.Json;
+using Neo.Network.RPC.Models;
+
+namespace Neo.Network.RPC
+{
+    public class StateApi : ContractClient
```

Comment:
Why inherit `ContractClient`? We don't invoke contract. Just use a `RpcClient` member?

Reply:
matches the pattern of `Nep17API` and `PolicyAPI` (though I just realized the name should be `StateAPI` not `StateApi`

Reply:
`Nep17` and `Policy` are contract related.